src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGridTests.cs

```diff
@@ -4068,6 +4068,60 @@ public class TestObject
         public string Property1 { get; set; }
     }
 
+    [WinFormsFact]
+    public void PropertyGrid_PropertyTabChangedEventTriggered()
+    {
+        using PropertyGrid propertyGrid = new();
+        int eventCallCount = 0;
+        PropertyTabChangedEventArgs eventArgs = null;
+
+        propertyGrid.PropertyTabChanged += (sender, e) =>
+        {
+            eventCallCount++;
+            eventArgs = e;
+        };
+
+        var tab1 = new TestPropertyTab();
+        var tab2 = new TestPropertyTab();
```

Comment:
Please use target type new
```suggestion
        TestPropertyTab tab1 = new();
        TestPropertyTab tab2 = new();
```